lib/services/hpcpackage_remain.pm

```diff
@@ -43,6 +43,8 @@ sub install_pkg {
     my @pkginstall = split('\n', script_output q[zypper search -r SLE-Module-HPC] . $version . q[-Pool -r SLE-Module-HPC] . $version . q[-Updates | cut -d '|' -f 2 | sed -e 's/ *//g' | grep -E '.*-hpc.*' | grep -vE 'system|module|suse' |  grep -vE '.*_[[:digit:]]+_[[:digit:]]+.*gnu|.*_[[:digit:]]+_[[:digit:]]+.*-hpc' | grep -vE '.*-static$' | grep -vE '.*hpc-macros.*'], proceed_on_failure => 1, timeout => 180);
     for (my $i = 1; $i < @pkginstall; $i = $i + 1) {
         zypper_call("in $pkginstall[$i]");
+        my $snapid= script_output(q{snapper list --disable-used-space 2>/dev/null | tail -n 1 | awk '{print $1}'});
```

Comment:
better add comments here why we need do this

Reply:
You need check whether the --disable-used-space is supported.

Reply:
OK